cnf-certification-test/suite_test.go

```diff
@@ -264,3 +353,209 @@ func incorporateVersions(claimData *claim.Claim) {
 		ClaimFormat:  ClaimFormatVersion,
 	}
 }
+
+var upgrader = websocket.Upgrader{
+	CheckOrigin: func(r *http.Request) bool {
+		return true
+	},
+}
+
+// Define an HTTP handler that triggers Ginkgo tests
+func runHandler(w http.ResponseWriter, r *http.Request) {
+	// Run Ginkgo tests
+	var requestData RequestData
+	//var responseData ResponseData
+	// Parse JSON data from the request body
+	err := json.NewDecoder(r.Body).Decode(&requestData)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+	// Create or open a log file
+	filename := "log.log"
+	if _, err := os.Stat(filename); err == nil {
+		// If it exists, truncate it to remove the contents
+		file, err := os.OpenFile(filename, os.O_TRUNC, 0)
+		if err != nil {
+			// Handle the error if necessary
+			panic(err)
+		}
+		file.Close()
+	}
+	logFile, err := os.OpenFile(filename, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0644)
+	if err != nil {
+		panic(err)
+	}
+	logrus.SetOutput(logFile)
+	originalStdout := os.Stdout
+	os.Stdout = logFile
+	// Read the YAML file
+	defer func() {
+		os.Stdout = originalStdout
+		logFile.Close()
+	}()
+	os.Setenv("KUBECONFIG", requestData.KubeConfigPath)
+	logrus.Infof("KUBECONFIG      : %v", requestData.KubeConfigPath)
+
+	logrus.Infof("Labels filter       : %v", requestData.SelectedOptions)
+
+	// Set the output of the logger to the log file
+
+	/*data, err := os.ReadFile("tnf_config.yml")
+	if err != nil {
+		log.Fatalf("Error reading YAML file: %v", err)
+	}
+
+	// Unmarshal the YAML data into a Config struct
+	var config configuration.TestConfiguration
+
+	err = yaml.Unmarshal(data, &config)
+	if err != nil {
+		log.Fatalf("Error unmarshaling YAML: %v", err)
```

Comment:
remove commented out code

Reply:
i need to move it another function thats the update of the tnf_config.yaml